vendor/modules.txt

```diff
@@ -41,6 +41,9 @@ github.com/Microsoft/hcsshim/internal/winapi
 github.com/Microsoft/hcsshim/osversion
 github.com/Microsoft/hcsshim/pkg/go-runhcs
 github.com/Microsoft/hcsshim/pkg/ociwclayer
+# github.com/Microsoft/hcsshim/test v0.0.0-20210227013316-43a75bb4edd3
+## explicit
+github.com/Microsoft/hcsshim/test/functional/manifest
```

Comment:
ugh. we will need to avoid `github.com/Microsoft/hcsshim/test/functional/manifest` as well